cdc/generation.cc

```diff
@@ -393,13 +393,11 @@ topology_description limit_number_of_streams_if_needed(topology_description&& de
     return topology_description(std::move(entries));
 }
 
-std::pair<utils::UUID, cdc::topology_description> make_new_generation_data(
+cdc::topology_description make_new_generation_description(
```

Comment:
I think there's no point in keeping this function now, I'd inline it (but do as you like here.)

Reply:
`topology_description_generator` is visible only in `generation.cc`, so I can't simply inline it in `storage_service::prepare_new_cdc_generation_data`. I could make it visible, but I think we can also remove this class completely. It's used only in `make_new_generation_description` in a trivial way. We could move the body of `topology_description_generator::generate` to `make_new_generation_description` and make `topology_description_generator::get_tokens` and `topology_description_generator::create_description` static functions that take no more than two arguments.

Reply:
Ok, if you wish, do the refactor. (It's optional though.)